compiler/one-cmds/one-infer

```diff
@@ -76,13 +72,56 @@ def _get_backends_list():
     return backends_list
 
 
+def _search_backend_driver(driver):
+    """
+    [one hierarchy]
+    one
+    ├── backends
+    ├── bin
+    ├── doc
+    ├── include
+    ├── lib
+    └── test
+
+    The list where `one-infer` finds its backend driver
+    - `bin` folder where `one-infer` exists
+    - `backends/**/bin/` folder
+
+    NOTE If there are drivers of the same name in different places,
+     the closer to the top in the list, the higher the priority.
+    """
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+
+    # CASE 0: check drvier name validation
+    if not driver:
+        return None
```

Comment:
When does this condition happen?

Reply:
From this source, never happened.. While designing the function structures, I tried to keep each function deals with every condition it can meet. And that makes this situation. 
Is it dropping the code quality to check the validity or threat twice or more dropping code quality?

Reply:
yeah, unnecessary condition'd better be removed. Actually, https://github.com/Samsung/ONE/pull/9158#discussion_r882327750 comment will change the overall codes.